train/model_functions.py

```diff
@@ -98,6 +98,9 @@ def create_gol_decoder():
 def create_patch_decoder():
   return create_decoder("patch_decoder")
 
+def create_grid_decoder():
+  return create_decoder("grid_decoder")
```

Comment:
I think it's simpler to remove these functions, and just have things like `decoder_grid = create_decoder("grid_decoder")` in train.py. what do you think?